core-tests/shared/src/test/scala/zio/FiberSpec.scala

```diff
@@ -119,7 +120,71 @@ object FiberSpec extends ZIOBaseSpec {
       testM("collectAll") {
         assertM(Fiber.collectAll(fibers).join)(anything)
       }
-    ) @@ sequential
+    ) @@ sequential,
+    suite("fiber dump tree")(
+      zio.test.test("render fiber hierarchy tree") {
```

Comment:
If I call just `test` instead of `zio.test.test` I get an error:
/zio/core-tests/shared/src/test/scala/zio/FiberSpec.scala:125:7: package zio.test is not a value

Reply:
Yes. This is an unfortunate side effect of the fact that `test` is both the package name and a method within the package. Fortunately this is only a problem within the `zio` package itself and I don't think there is a way to resolve other than renaming the package or the method, which I don't think we want to do, so we have to live with it. You handled it correctly.